docs/faq/network/blockchain-architecture.md

```diff
@@ -0,0 +1,103 @@
+
+### Are transactions cheaper than on other blockchains?
+
+Despite the type of network, transactions are usually cheaper on new blockchain networks due to the **reduced competition for block space**. Essentially, every block on a blockchain has a finite capacity for transactions. If the demand for transactions surpasses this capacity, users must compete for the limited space. This competition often manifests as transaction fees. On established networks like Ethereum, transaction fees can be pretty steep.
+
+On newer networks, like LUKSO, fewer users equate to **less competition and lower transaction fees**. Nevertheless, transaction fees are likely to increase as a network attracts more users and demand for block space rises over time.
+
+### What EVM Protocol Upgrades are live on LUKSO?
+
+The LUKSO Mainnet is up to date with [every Ethereum fork](https://ethereum.org/en/history/) until Shanghai, which was released in 2023. Except for Shapella (Capella and Shanghai), all upgrades were integrated within the mainnet launch.
+
+### Why is client diversity so important?
+
+Having a variety of clients in a blockchain network and improving our client diversity is critically important. Client diversity refers to utilizing different software clients in a blockchain network developed by various teams and in other programming languages to improve the following things:
+
+- **Security and Resilience**: Client diversity increases the robustness of the network. If there's a bug in one client, it doesn't bring down the entire network because other clients can continue to operate. This decentralization and redundancy is a fundamental aspect of blockchain security and resilience.
+- **Decentralization and Governanc**e: Client diversity promotes decentralization in the development and governance of the Ethereum network. It prevents any team or entity from having too much influence over the network's growth.
+
+For Ethereum, **client diversity has proven essential** in maintaining the network's robustness during several incidents. One of the most notable incidents were the Shanghai DoS Attacks in 2016, the OpenEthereum Consensus Bug in 2020, and the Prysm Client Incident in 2023. You can read more about them in the [extended client section](https://docs.luksoverse.io/docs/mainnet/complete-node-guide/blockchain-clients/client-setups) by the community.
+
+### How does Shapella enable validator withdrawals?
+
+Shanghai and Capella (Shapella) are the names of an Ethereum hard fork. Shanghai is the fork's name on the execution client side, and Capella is the upgrade name on the consensus layer client side. The included EIPs can be found [here](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/shanghai.md). The significant new features enable the **withdrawal of the validator's earnings and stake** from the network.
```

Comment:
"Shanghai and Capella (Shapella) are the names of an Ethereum hard fork" => "... is the name of ..."